app/code/Magento/InventoryAdminUi/Test/Mftf/Test/VerifyThatMessageOnly0LeftNotAppearedOnSimpleProductPageOnFrontendTest.xml

```diff
@@ -0,0 +1,211 @@
+                         after="adminCreateNewWebsite">
+                <argument name="website" value="{{NewWebSiteData.name}}"/>
+                <argument name="storeGroupName" value="{{NewStoreData.name}}"/>
+                <argument name="storeGroupCode" value="{{NewStoreData.code}}"/>
+            </actionGroup>
+            <actionGroup ref="AdminCreateStoreViewActionGroup" stepKey="adminCreateNewStoreView"
+                         after="adminCreateNewStore">
+                <argument name="StoreGroup" value="NewStoreData"/>
+                <argument name="customStore" value="customStore"/>
+            </actionGroup>
+            <selectOption selector="{{AdminEditStockSalesChannelsSection.websites}}" userInput="{{NewWebSiteData.name}}"
+                          stepKey="selectWebsiteAsSalesChannelForCustomStock"
+                          after="goToStockGridFilterResultsByInputEditStock1"/>
+            <magentoCLI command="config:set web/url/use_store 1"
+                        stepKey="displayOutOfStockYes" after="fillConfigValue"/>
+        </before>
+        <after>
+            <!--Delete website created in the before block -->
+            <actionGroup ref="AdminDeleteWebsiteActionGroup" stepKey="DeleteWebsite" after="deleteProduct">
+                <argument name="websiteName" value="{{NewWebSiteData.name}}"/>
+            </actionGroup>
+            <magentoCLI command="config:set web/url/use_store 0"
+                        stepKey="displayOutOfStockYes" after="fillConfigValue"/>
+        </after>
+        <!-- Add this product to second website -->
+        <click selector="{{ProductInWebsitesSection.sectionHeader}}" stepKey="openProductInWebsitesSection1"
+               after="goToProduct1"/>
+        <click selector="{{ProductInWebsitesSection.website(NewWebSiteData.name)}}" stepKey="selectSecondWebsite"
+               after="openProductInWebsitesSection1"/>
+        <remove keyForRemoval="setSimpleProductOutOfStockStatus"/>
+        <click selector="{{AdminProductSourcesGrid.rowDelete('0')}}" stepKey="deleteDefaultSourceFromCreatedProduct"
+               after="goToProduct1"/>
+        <actionGroup ref="AdminOnProductEditPageAssignSourceToProduct"
+                     stepKey="AdminOnProductEditPageAssignSourceToProduct1"
+                     after="deleteDefaultSourceFromCreatedProduct">
+            <argument name="filter_selector" value="AdminManageSourcesGridFilterControls.code"/>
+            <argument name="filter_value" value="$$createSource.source[source_code]$$"/>
+        </actionGroup>
+        <selectOption selector="{{AdminProductSourcesGrid.rowStatus('0')}}" userInput="Out of Stock"
+                      stepKey="setSimpleProductOutOfStockStatusToCustomStock"
+                      after="AdminOnProductEditPageAssignSourceToProduct1"/>
+        <remove keyForRemoval="openProductStorefront"/>
+        <amOnPage url="/{{customStore.code}}/$$createSimpleProduct.custom_attributes[url_key]$$.html"
```

Comment:
Please avoid hardcoding urls. You may create a page and use templates from it instead. You may check `app/code/Magento/Downloadable/Test/Mftf/Page/StorefrontCustomerDownloadableProductsPage.xml` and similar Pages for reference.